angr/engines/soot/statements/base.py

```diff
@@ -0,0 +1,80 @@
+
+import logging
+
+from archinfo.arch_soot import SootAddressDescriptor
+
+from ..exceptions import IncorrectLocationException
+from ..expressions import translate_expr
+from ..values import translate_value
+
+l = logging.getLogger('angr.engines.soot.statements.if')
+
+class SimSootStmt(object):
```

Comment:
`(object)` is not necessary for Python 3.

Reply:
Fixed Now